asllib/types.ml

```diff
@@ -91,42 +92,40 @@ let is_builtin_singular ty =
 let is_builtin_aggregate ty =
   match ty.desc with
   | T_Tuple _ | T_Array _ | T_Record _ | T_Exception _ -> true
-  | _ -> false 
-|: TypingRule.BuiltinAggregateType 
+  | _ -> false |: TypingRule.BuiltinAggregateType
 (* End *)
 
 (* Begin BuiltinSingularOrAggregate *)
-let is_builtin ty = is_builtin_singular ty 
-                  || is_builtin_aggregate ty 
-  |: TypingRule.BuiltinSingularOrAggregate
+let is_builtin ty =
+  is_builtin_singular ty
+  || is_builtin_aggregate ty |: TypingRule.BuiltinSingularOrAggregate
```

Comment:
please could we have the rule name on a different line for the display in latex not to overflow?